alns/select/MABSelector.py

```diff
@@ -170,12 +170,12 @@ def __call__(  # type: ignore[override]
         Returns the (destroy, repair) operator pair from the underlying MAB
         strategy
         """
-        try:
+        if self._mab._is_initial_fit:
             prediction = self._mab.predict(
                 contexts=self._context_extractor(curr)
             )
             return arm2ops(prediction)
-        except Exception:
+        else:
             # This can happen when the MAB object has not yet been fit on any
             # observations. In that case we return any feasible operator index
```

Comment:
I had to change this to use a conditional instead of exceptions for the control flow. Since the underlying exception raised by MABWiser is simply `Exception` and not a specific subclass, the original `try/except` block was catching any exception that was thrown while calling `is_initial_fit`, which is probably not what we want. (an alternative would be going back to the `try/except` method and checking the specific exception string)